akka-remote/src/main/scala/akka/remote/artery/ArterySubsystem.scala

```diff
@@ -57,6 +68,11 @@ private[remote] class ArterySubsystem(_system: ExtendedActorSystem, _provider: R
     else Future.successful(Done)
   }
 
+  def sendReply(to: Address, message: AnyRef) = {
```

Comment:
The purpose is to be able to send a reply from one of the inbound stages, therefore the method `replyTo` in `InboundContext`. Here it will probably be more clear when I collapse the `ArterySubsystem` and `Transport` classes.